src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -1,50 +1,149 @@
-          Sumission Guidelines
-        </div>
-        <div class="card-content" [innerHTML]="submissionGuidelines">
+<div class="challenge-card">
+  <div class="ev-card-panel ev-md-container card-bt-margin">
+    <p *ngIf="!isActive"><strong class="fw-semibold">Sorry, the challenge is not
+        active.</strong></p>
+    <div *ngIf="isActive" class="row row-lr-margin">
+      <div class="col-md-11 col-sm-11 col-lr-pad">
+        <span class="fs-20 fw-light">Submission Guidelines</span>
+        <div *ngIf="isChallengeHost">
+          <div class="fw-semibold content">
+            <p><strong class="fw-semibold">Note: </strong>Your submissions won't
+              be visible on leaderboard for
+              <strong class="fw-semibold blue-text"> Public Challenge Phases
+              </strong> since you are a challenge host
+              for this challenge.<br> This feature is only for your testing
+              purposes.</p>
+          </div>
         </div>
+        <div [innerHTML]="submissionGuidelines" class="content"></div>
       </div>
-      <div class="card card-light challenge-submit">
-        <div class="card-title">Please fill the following fields to submit</div>
-        <hr>
-        <div class="card-content">
-            <app-selectphase [phases]="filteredPhases" [phaseSelected]="phaseSelected()"></app-selectphase>
-        	<div class="phase-submissions" *ngIf="selectedPhase" >
-        		<div>
-              <span class="heading">Today's Remaining Submissions: </span>
-              <span class="content">{{selectedPhaseSubmissions['remaining_submissions_today_count']}}</span>
-            </div>
-            <div>
-              <span class="heading">Total Remaining Submissions: </span>
-              <span class="content">{{selectedPhaseSubmissions['remaining_submissions']}}</span>
+
+      <div *ngIf="isChallengeHost" class="col-md-1 col-sm-1 col-lr-pad">
+        <a class="pointer" (click)="editChallengePhase(phase)">
+          <i class="fa fa-pencil" aria-hidden="true">
+          </i>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="ev-card-panel">
+    <div class="bottom-card-container bottom-hr-line">
+      <strong class="fw-semibold content">
+        Please fill the following fields to submit
+      </strong>
+    </div>
+    <div class="row">
+      <div class="col-md-7 col-sm-7 col-xs-12">
+        <div class="exist-team-card">
+          <ul>
+            <b class="fw-bold content fs-14">Select phase:</b>
+            <app-selectphase [phases]="filteredPhases" [phaseSelectionType]="phaseSelectionType"
+              [phaseSelected]="phaseSelected()"></app-selectphase>
+
+            <div class="form">
+              <div class="row row-lr-margin">
+                <div class="col-md-10 col-sm-10 col-lr-pad">
+                  <app-input (change)="validateInput($event.target.value)"
+                    [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv,
```

Comment:
```suggestion
                    [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv, .gz'"
```
(Closing it in the same line)